src/components/form/HdRange.vue

```diff
@@ -0,0 +1,315 @@
+<template>
+  <div
+    :class="fieldClasses"
+    class="range field"
+  >
+    <input type="range"
+      :name="name"
+      ref="input"
+      :v-model="currentValue"
+      :required="required"
+      :disabled="disabled"
+      :min="minValue"
+      :max="maxValue"
+      :step="rangeStep"
+      @input="updateRangeDecoration"
+      @change="isLegacy ? updateRangeDecoration : null"
```

Comment:
`v-modal` handles this (with the edges cases that you mentioned in the other comment as well).
If you make the changes as I suggested in the other comment, watching `currentValue` will be enough.

Reply:
Let's sync on this.